docs/userguide.md

```diff
@@ -53,11 +54,24 @@ The results will appear in the box below. Select your preferred station, then pr
 
 When you're finished, close *Find Stations* to return to the main window.
 
+## Start/Stop streaming a station <a name = "part_start"></a>
+To start streaming a station, simply click on the little square on the bottom left corner of the station's icon. 
+
+The status bar should then indicate the name of the station you're currently playing.
+
+![Play](screenshots/04_Play_a_station.png)
+
+Click on the corresponding square on another station to switch to that station.
+
+To stop streaming, click on the square again.
+
+> **Note**: Stopping a stream this way may cause crashes in the current version of StreamRadio.
```

Comment:
Starting and stopping works here. Mentioning possible bugs doesn't really fit into a user guide. I'd remove this note, but add that double-clicking a station works also to start/stop streaming. This is much easier that aiming for a tiny icon...

Reply:
Interesting, because I haven't noticed that while learning how to use the app myself.

Reply:
On x86 it works ok for me too, so don't think it would be needed also